Corona-Warn-App/src/main/java/de/rki/coronawarnapp/presencetracing/risk/execution/PresenceTracingWarningTask.kt

```diff
@@ -89,7 +90,13 @@ class PresenceTracingWarningTask @Inject constructor(
 
             presenceTracingRiskRepository.reportCalculation(successful = true)
 
-            return Result(calculatedAt = nowUTC)
+            return Result()
+        }
+
+        val isPositive = coronaTestRepository.coronaTests.first().any { it.isPositive }
```

Comment:
Naming suggestion `isPositive` => `hasPositiveTest` ?

Reply:
Hm, feels wrong. We are asking the `CoronaTest` if it has a positive test? The `CoronaTest` IS the positive test :smile: